OptimizelySDK/Odp/OdpConfig.cs

```diff
@@ -117,5 +118,18 @@ public bool HasSegments()
         {
             return SegmentsToCheck?.Count > 0;
         }
+
+        /// <summary>
+        /// Determine equality between two OdpConfig objects
+        /// </summary>
+        /// <param name="toCompare">OdpConfig object to compare current instance against</param>
+        /// <returns>True if equal otherwise False</returns>
+        public bool Equals(OdpConfig toCompare)
```

Comment:
what's the reason to add this method. This will raise exception if ApiKey is null. Secondly use equals method where needed in this class and implement IEqu... interface as well.

Reply:
The goal of this is to control the equality determination of two ODP Configs. 

Thanks for the callout on null exception.  I'll change to use `IEqualityComparer`.

Reply:
Oh maybe `IEquatable<T>` might work instead of having a comparer. 🤔